sphere-models/src/it/java/io/sphere/sdk/types/commands/TypeCreateCommandTest.java

```diff
@@ -0,0 +1,43 @@
+package io.sphere.sdk.types.commands;
+
+import io.sphere.sdk.categories.Category;
+import io.sphere.sdk.models.TextInputHint;
+import io.sphere.sdk.test.IntegrationTest;
+import io.sphere.sdk.types.FieldDefinition;
+//important
```

Comment:
If you use `import io.sphere.sdk.types.*` then you don't get conflicts with the product attributes. We have there classes with the same name in different packages and the IDE suggests the wrong one.